contracts/EverlongStrategyFactory.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { EverlongStrategy } from "./EverlongStrategy.sol";
+import { IEverlongStrategy } from "./interfaces/IEverlongStrategy.sol";
+import { IEverlongStrategyFactory } from "./interfaces/IEverlongStrategyFactory.sol";
+import { EVERLONG_KIND, EVERLONG_VERSION } from "./libraries/Constants.sol";
+
+/// @author DELV
+/// @title EverlongStrategyFactory
+/// @notice A factory for creating Everlong instances.
+/// @custom:disclaimer The language used in this code is for coding convenience
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract EverlongStrategyFactory is IEverlongStrategyFactory {
+    /// @inheritdoc IEverlongStrategyFactory
+    string public constant name = "EverlongStrategyFactory";
```

Comment:
`name` shouldn't be a constant since it can be used to disambiguate deployments. This looks like it's duplicating `kind`.